packages/aws-cdk-lib/aws-apigateway/test/restapi.test.ts

```diff
@@ -379,29 +379,29 @@ describe('restapi', () => {
     // THEN
     expect(stack.resolve(api.url)).toEqual({
       'Fn::Join':
-    ['',
-      ['https://',
-        { Ref: 'apiC8550315' },
-        '.execute-api.',
-        { Ref: 'AWS::Region' },
-        '.',
-        { Ref: 'AWS::URLSuffix' },
-        '/',
-        { Ref: 'apiDeploymentStageprod896C8101' },
-        '/']],
+        ['',
```

Comment:
What are these new space changes? If this was a formatting change can you remove this?

Reply:
@sumupitchayan , do you think i should do something here ?